README.md

```diff
@@ -154,7 +154,7 @@ $ yarn
 That'll pull down the dependencies. Then run:
 
 ```bash
-$ gulp dev
+$ npm run dev
```

Comment:
Can you change this to `yarn run dev`? In theory it should be possible to develop Popcode without using `npm` at all.